clusterloader2/pkg/measurement/common/wait_for_pvcs.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"time"
+
+	"k8s.io/klog"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	defaultWaitForPVCsTimeout       = 60 * time.Second
+	defaultWaitForPVCsInterval      = 5 * time.Second
+	waitForBoundPVCsMeasurementName = "WaitForBoundPVCs"
+)
+
+func init() {
+	if err := measurement.Register(waitForBoundPVCsMeasurementName, createWaitForBoundPVCsMeasurement); err != nil {
+		klog.Fatalf("Cannot register %s: %v", waitForBoundPVCsMeasurementName, err)
+	}
+}
+
+func createWaitForBoundPVCsMeasurement() measurement.Measurement {
+	return &waitForBoundPVCsMeasurement{}
+}
+
+type waitForBoundPVCsMeasurement struct{}
+
+// Execute waits until desired number of PVCs are bound or until timeout happens.
+// PVCs can be specified by field and/or label selectors.
+// If namespace is not passed by parameter, all-namespace scope is assumed.
+func (w *waitForBoundPVCsMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
+	desiredPVCCount, err := util.GetInt(config.Params, "desiredPVCCount")
```

Comment:
is this the desired amt of PVC's created or bound? I feel like it might be bound in which case your should specify that

Reply:
I thought it would be intuitive because the name of the measurement is "WaitForBoundPVCs"

Reply:
understand the thought; however, it wasn't immediately clear to me so maybe not that intuitive :)

Reply:
hmm - it was intuitive for me based on the name. But if you have better suggestions for the name, I'm open.